src/js/services/forms.js

```diff
@@ -75,7 +75,12 @@ export default App.extend({
   fetchFormStoreSubmission({ submission }) {
     const channel = this.getChannel();
 
-    return $.when(Radio.request('entities', 'fetch:forms:definition', this.form.id)).then(definition => {
+    return $.when(
+      Radio.request(
+        'entities', 'fetch:forms:definition', this.form.id,
+        this.form.fetch(),
+      ),
+    ).then(definition => {
```

Comment:
Ah this is broken.  Not 100% sure why tests pass but this should be
`$.when(request, request)` not `$.when(Radio.request(request, request))` and the `then` would have `[definition]` because of the multiple requests

Reply:
:facepalm:

Reply:
Ha.. though I know why it works.  because `Radio.request` doesn't accept a 4th parameter.. so it isn't hurting anything.. and because it's not in the `$.when` the `definition` doesn't have to be wrapped in a `form`.. _but_ the `$.when` isn't waiting on the `form.fetch`

Reply:
Haha that explains it. Well, it should be fixed now :crossed_fingers: